package.json

```diff
@@ -21,7 +21,7 @@
     "tasks"
   ],
   "peerDependencies": {
-    "webpack": "^2.1.0-beta || ^2.2.0-rc || 2"
+    "webpack": "^2.1.0-beta || ^2.2.0-rc || 2 || 3"
```

Comment:
I could also allow the release candidate versions here if I change that to `^3.0.0-rc` but I'm not sure that's necessary now that `3.0.0` is out.